add note on allowing docker0 in firewalld (#1640)
hashicorp/terraform-website: content/source/docs/enterprise/before-installing/network-requirements.html.md

## Other Configuration

> 1. If a firewall is configured on the instance, be sure that traffic can flow out of the `docker0` interface to the instance's primary address. For example:

Reviewer: ```suggestion
1. If a firewall is configured on the instance, be sure that traffic can flow out of the `docker0` interface to the instance's primary address. 
```

>    *  To do this with UFW run: `ufw allow in on docker0`.

Reviewer: ```suggestion
   * To use UFW, run: `ufw allow in on docker0`.
```

>    *  To do this with firewalld run:

Reviewer: ```suggestion
   *  To use firewalld, run:
```

> 1. If a firewall is configured on the instance, be sure that traffic can flow out of the `docker0` interface to the instance's primary address. 
>    * To use UFW, run: `ufw allow in on docker0`.
>    *  To use firewalld, run:
> 
>         `firewall-cmd --permanent --zone=trusted --change-interface=docker0`
> 
>     This rule can be added before the `docker0` interface exists, so we recommend adding it before installing Docker.

Reviewer: One more tiny edit now that I see it all together - I promise I'm done after this :-) 

```suggestion
1. If a firewall is configured on the instance, run one of the following to allow traffic to flow out of the `docker0` interface to the instance's primary address. We recommend doing this before you install Docker.
   * To use UFW, run: `ufw allow in on docker0`
   *  To use firewalld, run: `firewall-cmd --permanent --zone=trusted --change-interface=docker0`
```

Author: No worries. As long as it becomes the best version of itself :)

>     This rule can be added before the `docker0` interface exists, so it is best to do it now, before the Docker installation.

Reviewer: I might also actually move this up to be right after the first sentence and before the examples. If we're making a recommendation, we should tell users before they stop reading because they've decided they don't want to do this right now.

```suggestion
    This rule can be added before the `docker0` interface exists, so we recommend adding it before installing Docker.
```
